examples/intel/file.py

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
one thing I think that would be nice is if we had a consistent log setup across 
- the SDK
- our examples

so people know how to debug things if anything goes wrong. maybe just setting up a simple logger in the examples would be a good place to start

Reply:
What do you mean with "consistent log setup"? Because I believe at this moment we should not have any log at all in SDK. 
Regarding to examples, I prefer to use simple prints in order just to show a little verbose response to user but no distract them setting up a logger and other stuff, I would like examples as simple as they can.

Reply:
I think the SDK should definitely push to some logger like "pangea" - it's useful for debugging.

examples is debatable, but I really think encouraging devs to set up logging in python is a good best practice. not enough people do it